repl/evaluator.go

```diff
@@ -430,7 +431,12 @@ type Evaluator struct {
 
 // NewEvaluator returns an inialized evaluator
 func NewEvaluator() (*Evaluator, error) {
-	env, err := cel.NewEnv()
+	env, err := cel.NewEnv(
+		ext.Strings(),
```

Comment:
It might be better to make this configurable since most clients are going to start with the standard env. Ok to defer, but update the README to note that these are enabled by default

Reply:
Done